parsers/jsonparser.cpp

```diff
@@ -0,0 +1,88 @@
+/*
+  This file is part of the Fairytale project
+
+  Copyright (C) 2018 M�rcio Pais

```

Comment:
This might need a larger discussion, but what do you think about me putting my name here, since it's a new file?